content/en/getting_started/workflow_automation/_index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Getting Started with Workflow Automation
+kind: documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/automate-end-to-end-processes-with-datadog-workflows/'
+      tag: 'Blog'
+      text: 'Automate End-to-End Processes with Workflow Automation'
+    - link: '/service_management/workflows/access/'
+      tag: 'Documentation'
+      text: 'Access and Authentication'
+    - link: '/security/cloud_security_management/workflows/'
+      tag: 'Documentation'
+      text: 'Automate Security Workflows'
```

Comment:
The single quotes in this section appear to work fine. We usually use double quotes, though. Can you change the single quotes to double ones?

Also, the titles should be in sentence case. For example, "Automate end-to-end processes with Workflow Automation".